.styleci.yml

```diff
@@ -12,3 +12,7 @@ disabled:
   - psr0
   - unalign_double_arrow
   - unalign_equals
+
+finder:
+  not-name:
+    - "*.php.twig"
```

Comment:
Shouldn't that be excluded by default anyway, because it doesn't match "*.php", it's "*.php*".